src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -15,8 +15,67 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
-    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
+    public static final String USERGUIDE_URL = "https://ay2122s2-cs2103t-t12-3.github.io/tp/UserGuide.html";
+    public static final String NEWLINE_AND_INDENTATION ="\n     ";
+    public static final String HELP_MESSAGE = "Tracey supports the following features: Find, Add, Delete, Edit, Clear, "
+        + "Summarise, List, Help and Filter.\n"
+        + "This guide aims to showcase the syntax used for the aforementioned features.\n"
+        + "For a full comprehensive guide, please refer to " + USERGUIDE_URL
+        + "\n"
+
+        + "\n1. FIND a particular contact in Tracey."
+        + NEWLINE_AND_INDENTATION
+        + "Format: find NAME"
+        + NEWLINE_AND_INDENTATION
+        + "Example 1: find John\n"
+
+        + "\n2. ADD a student with relevant details into Tracey."
+        + NEWLINE_AND_INDENTATION
+        + "Format: add n/NAME p/PHONENUMBER e/EMAIL ..."
+        + NEWLINE_AND_INDENTATION
+        + "Example 1: add n/Melvin f/SOC cs/ Negative"
+        + NEWLINE_AND_INDENTATION
+        + "Example 2: add e/student69@u.nus.edu n/ Martin\n"
+
+        + "\n3.DELETE a contact at a specific INDEX. The index refers to the index number shown in the " +
+            "displayed person list."
+        + NEWLINE_AND_INDENTATION
+        + "Format: delete INDEX"
+        + NEWLINE_AND_INDENTATION
+        + "Example 1: Delete 2"
+        + NEWLINE_AND_INDENTATION
+        + "Example 2: Delete 4\n"
+
+        + "\n4. EDIT the person at the specified INDEX. The index refers to the index number shown in the " +
+            "displayed person list."
+        + NEWLINE_AND_INDENTATION
+        + "Format: edit INDEX n/NAME ..."
+        + NEWLINE_AND_INDENTATION
+        + "Example 1: edit 1 p/91234567 e/johndoe@example.com"
+        + NEWLINE_AND_INDENTATION
+        + "Example 2: edit 2 n/David Limpeh t/\n"
+
+        + "\n5. CLEAR all the data inside Tracey"
```

Comment:
Maybe can include a confirmation pop-up will appear or a warning that says cleared data cannot be retrieved